lib/foreman_remote_execution_core/script_runner.rb

```diff
@@ -146,6 +146,7 @@ def ssh_options
       ssh_options[:paranoid] = true
       ssh_options[:auth_methods] = available_authentication_methods
       ssh_options[:user_known_hosts_file] = prepare_known_hosts if @host_public_key
+      ssh_options[:verbose] = settings[:ssh_log_level]
```

Comment:
While testing, I've noticed that the ssh uses a `Logger` around `STDERR`, if not provided with different `:logger` option: therefore, I was not seeing the log messages from SSH in the logs.

I see that using the existing logger we have directly is probably not what we want, as even if we are interested into
`info`, or even `debug` log messages from proxy, we might not need the `debug` messages from the ssh.

What about having a logger wrapper around our own logger, to pass the messages to the logger only if the level is >= as `ssl_log_level`. This would require the standard logger to be configured more verbosely that the ssl logger, to get the messages propagated: I guess we could just refuse to start if the `:ssl_log_level > logger.level`.